client/me/help/help-courses/course.jsx

```diff
@@ -25,10 +26,15 @@ export default localize( ( props ) => {
 			<Card compact>
 				<h1 className="help-courses__course-title">{ title }</h1>
 				<p className="help-courses__course-description">{ description }</p>
+				{ ! isBusinessPlanUser &&
+					<HelpTeaserButton
+						href="/plans"
```

Comment:
This URL might need a `siteSlug`. Currently it redirects to the select a site page:
![screen shot 2016-08-25 at 10 21 47 pm](https://cloud.githubusercontent.com/assets/416133/17968854/58d303de-6b12-11e6-9c0b-202f9c782eda.png)